docker/vast/docker-compose.dev.yaml

```diff
@@ -0,0 +1,6 @@
+version: "3"
```

Comment:
what about calling this `docker-compose.build.yaml`? no strong opinion about it though